tests/integ/modin/series/test_at.py

```diff
@@ -129,3 +129,12 @@ def test_at_neg(
 ):
     with pytest.raises(KeyError):
         default_index_snowpark_pandas_series.at[key]
+
+
+@sql_count_checker(query_count=0)
+def test_raise_set_cell_with_list_like_value_error():
```

Comment:
Nit: Any particular reason this test is added under the `series` directory, while the three others under the `frame` directory?

I'd suggest either add all of them under one directory (say `frame`) or perhaps better to keep the dataframe parts of the tests under `frame` and the series parts under `series`.

Reply:
oh, I should put under `frame`. Good catch!